docs/docs/20-usage/90-pipeline-management.md

```diff
@@ -0,0 +1,73 @@
+# Advanced pipeline management
+
+## Using variables
+Once your pipeline starts to grow in size, it will become important to keep it DRY ("Don't Repeat Yourself") by using variables and environment variables. Depending on your specific need, there are a number of options.
```

Comment:
Please add empty newline between section title and content
```suggestion

Once your pipeline starts to grow in size, it will become important to keep it DRY ("Don't Repeat Yourself") by using variables and environment variables. Depending on your specific need, there are a number of options.
```